content/accounts.md

```diff
@@ -0,0 +1,702 @@
+
+First, figure out which template you want to replace by looking at the source code of the package. For example, in the `useraccounts:unstyled` package, the templates are listed [in this directory on GitHub](https://github.com/meteor-useraccounts/unstyled/tree/master/lib). By squinting at the file names and looking for some of the HTML strings, we can figure out that we might be interested in replacing the `atPwdFormBtn` template. Let's take a look at the original template:
+
+```html
+<template name="atPwdFormBtn">
+  <button type="submit" class="at-btn submit {{submitDisabled}}" id="at-btn">
+    {{buttonText}}
+  </button>
+</template>
+```
+
+Once you've identified which template you need to replace, define a new template. In this case, we want to modify the class on the button to work with the CSS for the rest of the app. Make sure that it renders the helpers you need just like the original, for example `buttonText`:
+
+```html
+<template name="override-atPwdFormBtn">
+  <button type="submit" class="btn-primary">
+    {{buttonText}}
+  </button>
+</template>
+```
+
+Then, use the `replaces` function on the template to override the existing template from `useraccounts`:
+
+```js
+Template['override-atPwdFormBtn'].replaces('atPwdFormBtn');
+```
+
+<h3 id="useraccounts-customizing-routes">Customizing routes</h3>
+
+In addition to having control over the templates, you'll want to be able to control the routing and URLs for the different views offered by `useraccounts`. Since Flow Router is the officially recommended routing option for Meteor, we'll go over that in particular.
+
+First, we need to configure the layout we want to use when rendering the accounts templates:
+
+```js
+AccountsTemplates.configure({
+  defaultTemplate: 'authPage',
+  defaultLayout: 'appBody',
+  defaultContentRegion: 'main',
+  defaultLayoutRegions: {}
+});
+```
+
+In this case, we want to use the `appBody` layout template for all of the accounts-related pages. This template has a content region called `main`. Now, let's configure some routes:
+
+```js
+AccountsTemplates.configureRoute('signIn', {
+  name: 'signin',
+  path: '/signin'
+});
+
+AccountsTemplates.configureRoute('signUp', {
+  name: 'join',
+  path: '/join'
+});
+
+AccountsTemplates.configureRoute('forgotPwd');
+
+AccountsTemplates.configureRoute('resetPwd', {
+  name: 'resetPwd',
+  path: '/reset-password'
+});
+```
+
+Now, we can easily render links to our login page like so:
+
+```html
+<div class="btns-group">
+  <a href="{{pathFor 'signin'}}" class="btn-secondary">Sign In</a>
+  <a href="{{pathFor 'join'}}" class="btn-secondary">Join</a>
+</div>
+```
+
+Note that we have specified a password reset route. For this URL to be sent in Meteor's password reset email, we need to configure it:
+
+```js
+Accounts.urls.resetPassword = (token) => {
+  return Meteor.absoluteUrl(`reset-password/${token}`);
+};
+```
```

Comment:
actually, URL configuration is already managed automatically. See [these lines](https://github.com/meteor-useraccounts/flow-routing/blob/master/lib/core.js#L105-L110) for the example above.

You could turn the sentence into something like: _usually you should configure the URL this way, but using `useraccounts` routing packages you don't have to worry about it..._